apps/ltn/src/select_boundary.rs

```diff
@@ -154,7 +154,7 @@ impl SelectBoundary {
     }
 
     // Ok(Some(x)) means the current neighbourhood was destroyed, and the caller should switch to
-    // focusing on a different neigbhorhood
+    // focusing on a different neighborhood
```

Comment:
Oof, I'm still not consistent with the American/UK spelling. I'll make another pass over this later